app/poros/animal_detail.rb

```diff
@@ -4,14 +4,24 @@ class AnimalDetail
               :prey, 
               :predators, 
               :habitat, 
-              :scientific_name
+              :scientific_name,
+              :fun_fact,
+              :top_speed,
+              :life_span,
+              :weight,
+              :diet
 
   def initialize (animal_data, photo_data)
     @name = animal_data[:name]
     @photo_url = photo_data[:photos].first[:url]
     @prey = (animal_data.dig(:characteristics, :main_prey) || animal_data.dig(:characteristics, :prey))
     @predators = animal_data.dig(:characteristics, :predators)
     @habitat = animal_data.dig(:characteristics, :habitat)
+    @fun_fact = (animal_data.dig(:characteristics, :slogan) || "Do not pet this animal in the wild!")
+    @top_speed = animal_data.dig(:characteristics, :top_speed)
```

Comment:
Can't believe this works, this was fun figuring this out without writing method!!!